src/server/my-space-controller.js

```diff
@@ -0,0 +1,47 @@
+import PageRenderer from './page-renderer.js';
+import { PAGE_NAME } from '../domain/page-name.js';
+import RoomService from '../services/room-service.js';
+import ServerConfig from '../bootstrap/server-config.js';
+import { FEATURE_TOGGLES } from '../common/constants.js';
+import ClientDataMapper from '../server/client-data-mapper.js';
+import needsAuthentication from '../domain/needs-authentication-middleware.js';
+
+class UserController {
+  static get inject() { return [ServerConfig, PageRenderer, RoomService, ClientDataMapper]; }
+
+  constructor(serverConfig, pageRenderer, roomService, clientDataMapper) {
+    this.serverConfig = serverConfig;
+    this.roomService = roomService;
+    this.pageRenderer = pageRenderer;
+    this.clientDataMapper = clientDataMapper;
+  }
+
+  async handleGetMySpacePage(req, res) {
+    const { user } = req;
+
+    let rooms = [];
+    if (!this.serverConfig.disabledFeatures.includes(FEATURE_TOGGLES.rooms)) {
```

Comment:
This would be better put into the "register pages". It was done like that the room-controller.

Reply:
It's not the same thing, we do want the registration to happen in this case, we just bypass the data fetching.

Reply:
Ah, ok